$OpenBSD: patch-src_doc_CMakeLists_txt,v 1.1 2020/11/19 12:07:59 sthen Exp $

check whether we're going to build iv, don't check if it's already installed.
part of

From 33f19f3da1e5e01aa63c711d2fd3ee7ff9310248 Mon Sep 17 00:00:00 2001
From: Richard <hobbes1069@gmail.com>
Date: Mon, 5 Mar 2018 11:38:29 -0600
Subject: [PATCH] Make building and installing of documentation more packaging
 friendly. (#1890)

Index: src/doc/CMakeLists.txt
--- src/doc/CMakeLists.txt.orig
+++ src/doc/CMakeLists.txt
@@ -19,8 +19,7 @@ if (UNIX AND TXT2MAN AND PYTHONINTERP_FOUND)
     message (STATUS "Unix man page documentation will be generated")
     set (cli_tools oiiotool iinfo maketx idiff igrep iconvert)
 
-    find_program (IV_FOUND iv)
-    if (IV_FOUND)
+    if (TARGET iv)
         list (APPEND cli_tools iv)
     endif()
 
